### PR TITLE
fix of type issue in class combo disable

### PR DESCRIPTION
**Pull request by dkhnn**

### CODE REVIEW

File: src/app/pages/user/dialogs/add/add-user.dialog.html [hunk 1]
```diff
@@ -59,9 +59,9 @@ <h3 mat-dialog-title>הוספת משתמש חדש</h3>
               <mat-error *ngIf="EmailFormControl.hasError('email')">אימייל לא חוקי</mat-error>
                 </mat-form-field>
               </div>
-              <div class="msw-form-row">
+              <div clSass="msw-form-row">
```
Comment (eran1205) [inline src/app/pages/user/dialogs/add/add-user.dialog.html]: What is clSass? typo?

File: src/app/pages/user/dialogs/add/add-user.dialog.html [hunk 2]
```diff
@@ -59,9 +59,9 @@ <h3 mat-dialog-title>הוספת משתמש חדש</h3>
...
                   <mat-form-field color="primary">
-                    <mat-select placeholder="כיתה" formControlName="class" [(ngModel)]="data.Class" [(value)]="data.Class" name="class" [disabled]="currentRole !== userRoleEnum.TEACHER">
+                    <mat-select placeholder="כיתה" formControlName="class" [(ngModel)]="data.Class" [(value)]="data.Class" name="class" [disabled]="isManager()">
```
Comment (dimkinv) [inline src/app/pages/user/dialogs/add/add-user.dialog.html]: Maybe instead it's better to create service that had generic function `isInRole(role: Role Type)`?

Comment (dkhnn) [reply src/app/pages/user/dialogs/add/add-user.dialog.html]: not sure how this is DRY. I removed the method and just added comparison in HTML

**Comment by dkhnn** [comment]
@eran1205 do your thing

**Pull request merged**